Userland/Applications/Calculator/main.cpp

```diff
@@ -51,20 +52,22 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         auto clipboard = GUI::Clipboard::the().fetch_data_and_type();
         if (clipboard.mime_type == "text/plain") {
             if (!clipboard.data.is_empty()) {
-                widget->set_entry(KeypadValue(StringView(clipboard.data)));
+                widget->set_entry(Crypto::BigFraction(StringView(clipboard.data)));
             }
         }
     }));
 
     auto& constants_menu = window->add_menu("&Constants");
+    auto const power = Crypto::NumberTheory::Power("10"_bigint, "10"_bigint);
```

Comment:
Doesn't that change the constant's values further down?

Reply:
In what way?
I can't re-test it now, but I remember testing it.